docs/.vuepress/components/HomeSponsors.vue

```diff
@@ -27,11 +27,17 @@ export default {
 <style>
 #sponsors {
   text-align: center;
-  padding: 35px 40px 45px;
-  margin: 0 -2rem;
+  padding: 35px 30px 45px;
   background-color: #f6f6f6;
 }
 
+@media (min-width: 768px) {
+  #sponsors {
+    padding: 35px 40px 45px;
+    margin: 0 -2rem;
```

Comment:
I removed the negative left and right margin because it was generating:
- an horizontal scrollbar on mobile
- and a margin issue on the top navbar (that is present on the production website, but it became even more pronounced when adding the banner).

To replicate the issue, visit the site on mobile and scroll down. You'll see that the top margin of the navbar gets collapsed. And scroll to the sides, you will see the website has an horizontal scrollbar.

Here's a screenshot: https://imgur.com/a/v7jMwEY

Feel free to ignore this fix if you don't think is appropriate. Doing so will cut some pixels from the top of the banner.

Let me know if you have questions.

Reply:
I see, thanks, let's keep it then?